go/kbfs/libfuse/errors.go

```diff
@@ -75,5 +76,10 @@ func filterError(err error) error {
 	case libkbfs.RevGarbageCollectedError:
 		return errorWithErrno{err, syscall.ENOENT}
 	}
+
+	if os.IsNotExist(errors.Cause(err)) {
```

Comment:
Do we wanna add `libkbfs. NoSuchNameError` above too just in case?

Reply:
libfuse/libdokan/libfs all handle that specifically right now, but sure I guess it doesn't hurt to add this here too.